### ISSUE TITLE
[Bug] Find a Form - 21-0790 Details Page Should Not Be Published

### ISSUE THREAD

**Issue by johnhashva** [department-of-veterans-affairs/va.gov-team#18920]
## What happened?

As reported by Jen Lee:

- The VA Form 21-0790 was retired by the compensation business line. It appears to be now removed from the VA forms database – e.g., when you search for it,  you get the ‘retired/removed’ message: https://www.va.gov/find-forms/?q=21-0790
- However, I’m still seeing that the form detail landing page is still published: https://www.va.gov/find-forms/about-form-21-0790/   This seems to be a bug because when a PDF form is removed from the source VA forms DB, the corresponding form detail page is supposed to become unpublished (archived).

In the CMS, it looks like the last date of the CMS migration was 10/21/20. But the business likely removed this form from the DB recently (at least some time after my email of Dec 30 to them).

My understanding per the way Steve Wirt set the CMS up for us with forms is that when the form is removed in the forms DB, that change gets migrated as a “Deleted:True” meta data into the CMS during nightly migration. And that “Deleted:True” flag automatically unpublishes the form detail page and it goes into the “archived” state.

Visual Reference:

<img width="794" alt="Screen Shot 2021-01-23 at 10 08 36 AM" src="https://user-images.githubusercontent.com/63107147/105605854-635bb780-5d64-11eb-964d-e61cc20fd5dd.png">

<img width="1059" alt="Screen Shot 2021-01-23 at 10 08 48 AM" src="https://user-images.githubusercontent.com/63107147/105605857-65be1180-5d64-11eb-83e7-331afafbae36.png">

## Specs:
- Device: All
- Browser: All
- Test User _(if applicable)_:

## Steps to Reproduce
- URL: https://www.va.gov/find-forms/?q=21-0790
- Test User _(if applicable)_:

## Desired behavior
_ The forms detail page for 21-0790 -- https://www.va.gov/find-forms/?q=21-0790 -- should now longer be published/accessible 

## Acceptance Criteria
- [x] When I search for VA form 21-0790 whether on VA.gov or Google search, [the forms detail page](https://www.va.gov/find-forms/about-form-21-0790/) should no longer be displayed as a search result   **OR do we instead want to put the same _The form you're looking for has been retired or is no longer valid, and has been removed from the VA forms database._ on that page (e.g. this would be the preferred convention/rule we want for all retired/removed VA forms)**

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Bug`**

**Comment by johnhashva** [comment]
@MarciMcGuireGCIO the deleted form bug Jen reported via email -- worth a convo about this one next week as its a use case we want to make sure to have solutioned.

**Comment by MarciMcGuireGCIO** [comment]
Thank you for capturing this, @johnhashva .

**Comment by MarciMcGuireGCIO** [comment]
Posted on Slack https://dsva.slack.com/archives/CBU0KDSB1/p1611590647138900

**Comment by MarciMcGuireGCIO** [comment]
@swirtSJW has opened a ticket for CMS https://dsva.slack.com/archives/CUB5X5MGF/p1611671883007400, so I am closing our ticket, but will continue to monitor.

**Comment by MarciMcGuireGCIO** [comment]
Keeping this open to monitor until it's remediated on CMS.

**Comment by NCHLiberty** [comment]
Team Matsumoto (Lighthouse Forms API team) has created this Jira ticket corresponding to this GitHub issue. https://vajira.max.gov/browse/API-5083

**Comment by MarciMcGuireGCIO** [comment]
:wave: Hi @NCHLiberty - just doing a sprint wrap-up check on the status of open issues.  Are there any updates on this?  Thanks!

**Comment by MarciMcGuireGCIO** [comment]
Hi @NCHLiberty; checking open issues on our board.  Any updates on this one?  Thank you!

**Comment by NCHLiberty** [comment]
hi @MarciMcGuireGCIO this one did not fit in the sprint that is ending now but is prioritized into the one that is about to start.

**Comment by MarciMcGuireGCIO** [comment]
@NCHLiberty Thank you for the update!

**Comment by NCHLiberty** [comment]
This ticket is prioritized into the sprint that begins tomorrow.

**Comment by mviterna** [comment]
@MarciMcGuireGCIO @NCHLiberty @johnhashva  Did this get resolved? I can't access the form via https://www.va.gov/find-forms/about-form-21-0790/  or did I misunderstand the ticket? It could be that the timeout fix we put in resolved this. I could potentially see where some nightly migrations would time out and some of these random issues pop up. But if there is another example form for me to look at, let me know

**Comment by MarciMcGuireGCIO** [comment]
@mviterna this seems to be resolved! @johnhashva can you also confirm whenever you're able?

**Comment by johnhashva** [comment]
@mviterna and @MarciMcGuireGCIO looks resolved to me as well -- e.g. the form detail page is now longer available, I'm getting page not found.

**Comment by mviterna** [comment]
@johnhashva @MarciMcGuireGCIO  I wish all support tickets were that easy! I'll close the team matsumoto jira ticket.

**Comment by MarciMcGuireGCIO** [comment]
If only! Lol! Thank you!

**Comment by swirtSJW** [comment]
Here is the related CMS ticket and all the possible scenarios I proposed that might have led to this one off misalignment of the data. The only remaining possibilities are untestable with what was in place at the time that this occurred.  I added some additional tracking so that if this ever occurs again we can rule in or out the last two  scenarios.

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4083
